### PR TITLE
Update `aliquot` to new version 2.1.1

### PR DESCRIPTION
**Pull request by cbobach**
Update `README.md` with description on how to push gems with cyclic dependency to rubygems.org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,36 @@ a.process
...
 
-To be sure that unit tests run properly, you can run them in a docker container.
+To be sure that unit tests run properly, you can run them in a Docker container.
 
 ```bash
 docker run -ti --rm -v $(pwd):/opt/aliquot ruby:2.7.4 bash
 cd /opt/aliquot
 bundle install
 bundle exec rspec
+exit
```
Comment (mt-clearhaus) [inline README.md:27]: Feels a little superfluous :smile:

Comment (mt-clearhaus) [reply README.md:27]:
```suggestion
```

File: aliquot.gemspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'excon',          '~> 0.71.0'
   s.add_runtime_dependency 'hkdf',           '~> 0.3'
 
-  s.add_development_dependency 'aliquot-pay', '~> 2.1.0'
+  s.add_development_dependency 'aliquot-pay', '~> 2.1.1'
   s.add_development_dependency 'rspec',       '~> 3'
-  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry',         '~> 0.14.1'
```
Comment (mt-clearhaus) [inline aliquot.gemspec:23]: Isn't it superfluous to specify a specific version of `pry`?

```suggestion
  s.add_development_dependency 'pry'
```

**Review by mt-clearhaus — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (kbadk) [reply README.md:27]: Keep in mind that these instructions are written inside the container, so `exit` is to put you back into your local shell. Kind of superfluous, but also something you'll inevitably be writing afterwards (unless you close the window or CTRL+D).

Comment (mt-clearhaus) [reply README.md:27]: Your sentence can be read both as an argument for keeping it and for removing it :slightly_smiling_face: 

It's not important to me whether or not we keep it.

File: aliquot.gemspec (hunk 2 above)
Comment (cbobach) [reply aliquot.gemspec:23]: This is to get rid of this warning `gem buld` throws.
```console
➜  aliquot git:(aliquot_2.1.1) ✗ gem build
WARNING:  open-ended dependency on pry (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  See https://guides.rubygems.org/specification-reference/ for help
```

Comment (mt-clearhaus) [reply aliquot.gemspec:23]: Cool :+1:

**Pull request merged**